### ISSUE TITLE
call is ambiguous between the following methods or properties: 

### ISSUE THREAD

**Issue by kudlatiger** [RSuter/NSwag#1843]
I am getting below error at app.UseSwagger();

> call is ambiguous between the following methods or properties: 'Microsoft.AspNetCore.Builder.SwaggerBuilderExtensions.UseSwagger(Microsoft.AspNetCore.Builder.IApplicationBuilder, System.Action<Swashbuckle.AspNetCore.Swagger.SwaggerOptions>)' and 'Microsoft.AspNetCore.Builder.NSwagApplicationBuilderExtensions.UseSwagger(Microsoft.AspNetCore.Builder.IApplicationBuilder, System.Action<NSwag.AspNetCore.SwaggerDocumentMiddlewareSettings>)'

**Comment by RSuter** [comment]
You are probably using swashbuckle and nswag at the same time - either you remove one or you call the extension method fully qualified

**Comment by kudlatiger** [comment]
Yes you are right. I have removed swashbuckle and I still see the error

**Comment by danshoubs** [comment]
I am also having a similar problem, for a number or reasons I'm trying to move to swashbuckle, but removing the NSwag packages doesn't seem to be enough to get rid of NSwag. I'm not sure what I'm missing, I've removed the nuget package, deleted bin and obj files, cleaned an rebuilt, recloned etc!

`CS0121	The call is ambiguous between the following methods or properties: 'Microsoft.AspNetCore.Builder.NSwagApplicationBuilderExtensions.UseSwagger(Microsoft.AspNetCore.Builder.IApplicationBuilder, System.Action<NSwag.AspNetCore.SwaggerDocumentMiddlewareSettings>)' and 'Microsoft.AspNetCore.Builder.SwaggerBuilderExtensions.UseSwagger(Microsoft.AspNetCore.Builder.IApplicationBuilder, System.Action<Swashbuckle.AspNetCore.Swagger.SwaggerOptions>)'`
